src/seed_data/factories/Course.factory.ts

```diff
@@ -0,0 +1,43 @@
+import { define } from 'typeorm-seeding';
+import { Course } from '../../entity/Course';
+import { subjects, getRandomIntInclusive, toUpperCase } from '../utils'
+
+import * as faker from 'faker';
+
+faker.seed(123);
+
+let courseNumbers = [];
+
+function generateCourseNumber() {
+    let num = faker.random.number();
+    if (num < 1000) {
```

Comment:
would it be better to pass in an object into number() like this?
```
{ min: 1000 }
``` 
or 
```
{ min: 1000, max: 9999 }
```
https://github.com/Marak/faker.js/wiki/Random

Reply:
oh yes! thanks for pointing this out -- the faker library is v large and i missed this